bigbluebutton_api_python/bigbluebutton.py

```diff
@@ -127,6 +127,13 @@ def set_config_xml(self, meeting_id, xml):
                                                                            "meetingID": meeting_id})
         return SetConfigXMLResponse(response)
 
+    def _parse_response(self, response):
+        # parse response to json if json else parse as xml
+        try:
```

Comment:
the right way to do that, is by checking the response format from the response header metadata, there is an information about if this is a json or xml or whatever